Firestore/CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 - [fixed] Fixed a regression in the Firebase iOS SDK release 4.11.0 that could
   cause `getDocument()` requests made while offline to be delayed by up to 10
   seconds (rather than returning from cache immediately).
+- [feature] Added a new `Timestamp` class to represent timestamp fields,
+  currently supporting up to microsecond precision. It can be passed to API
+  methods anywhere a system Date is currently accepted. To make
+  `DocumentSnapshot`s read timestamp fields back as `Timestamp`s instead of
+  Dates, set the newly-added flag `areTimestampsInSnapshotsEnabled` to `true` in
```

Comment:
Optional suggestions:
* "set" => "you can set"
* move "to true" after "in FirestoreSettings"

Reply:
Done, thanks!